### PR TITLE
Allow mulitple OR calculations to be done on Question validation

### PR DESCRIPTION
**Pull request by dwyeradam**
Allow for multiple calculation validation to be performed on a single question
Include value as an option to use instead of answer_id
Apply change to QBS (2_0001.json)

### CODE REVIEW

File: scripts/test_schemas.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+docker pull onsdigital/eq-schema-validator:multi-rule-calculation
```
Comment (jonnyshaw89) [inline scripts/test_schemas.sh]: This needs revert.

File: tests/app/forms/test_questionnaire_form.py [hunk 2]
```diff
@@ -124,8 +124,8 @@ def test_invalid_calculation_type(self):
             question_json = {
                 'id': 'breakdown-question',
                 'type': 'Calculated',
-                'calculated': {
-                    'calculated_type': 'subtraction',
+                'calculations': [{
+                    'calculation_type': 'subtraction',
```
Comment (jonnyshaw89) [inline tests/app/forms/test_questionnaire_form.py:128]: Whats this?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: tests/app/forms/test_questionnaire_form.py (hunk 2 above)
Comment (dwyeradam) [reply tests/app/forms/test_questionnaire_form.py:128]: So it appears @shahi645 added tests for scenarios he doesn't allow at a schema level and we let it through. @shahi645 do we need to add this at schema level or remove this test until we have a full set of tests for it?

Comment (dwyeradam) [reply tests/app/forms/test_questionnaire_form.py:128]: Ignore this. The test is actually ensuring it can handle an invalid calculation type.

Comment (shahi645) [reply tests/app/forms/test_questionnaire_form.py:128]: it is at schema level but Its good to test it here too, it was a PR change requested

**Review by jonnyshaw89 — APPROVED** [review]

**Review by shahi645 — APPROVED** [review]
LGTM 
Only thing is the order of multi rules need to be explicit, maybe something we want to look at in the future

**Pull request merged**